config/m-nt.h

```diff
@@ -63,3 +63,22 @@
 #endif
 
 #define FLAT_FLOAT_ARRAY
+
+#if defined(_MSC_VER) && _MSC_VER < 1600
```

Comment:
This guard is unnecessary (and it should be `1200`, not `1600`: Visual Studio 2010 added `stdint.h`, but these `typedef`s existed since Visual Studio .NET 2002 in, amongst others, `stdarg.h` and `stddef.h`)